### ISSUE TITLE
https://hpo.jax.org/app/browse/term/HP:0006957

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7703]
**HPO term**
Loss of ability to walk HP:0006957

**Reason for deprecation**
synonymous with Loss of ambulation HP:0002505

**Suggested replacement or term to consider**

**Comment by pnrobinson** [comment]
done
